### PR TITLE
Update ntfx adapter to include inventory

### PR DESCRIPTION
**Pull request by g1nt0ki**

### CODE REVIEW

**Comment by llamatester** [comment]
The adapter at projects/nftx exports TVL: 
        
 
 
    ethereum                  42.13 M
    
    total                    42.13 M

File: projects/nftx/index.js [hunk 1]
```diff
@@ -17,14 +18,45 @@ async function tvl(_timestamp, block) {
...
+  const token_balances = await sdk.api.abi.multiCall({
+    abi: 'erc20:totalSupply',
+    calls: Array.from(tokens).map(token => ({
+      target: token,
+    })),
+    block
+  })
+
   const balances = {}
+  token_balances.output.forEach(item => sdk.util.sumSingleBalance(balances, item.input.target, item.output))
```
Comment (0xngmi) [inline projects/nftx/index.js:30]: sdk.util.sumBalances()?

**Comment by llamatester** [comment]
The adapter at projects/nftx exports TVL: 
        
 
 
    ethereum                  42.20 M
    
    total                    42.20 M

**Comment by llamatester** [comment]
The adapter at projects/mahadao exports TVL: 
        
 
 
    bsc-pool2                 2.63 k
    ethereum                  73.8981
    bsc                       429.83 k
    ethereum-pool2            178.34 k
    polygon-staking           1.04 M
    polygon                   528.83 k
    polygon-pool2             184.43 k
    pool2                     365.40 k
    staking                   1.04 M
    
    total                    958.74 k

**Comment by llamatester** [comment]
The adapter at projects/nftx exports TVL: 
        
 
 
    ethereum                  30.58 M
    
    total                    30.58 M

**Pull request merged**